Mathlib/Topology/Homeomorph.lean

```diff
@@ -77,6 +77,8 @@ protected def symm (h : α ≃ₜ β) : β ≃ₜ α where
   toEquiv := h.toEquiv.symm
 #align homeomorph.symm Homeomorph.symm
 
+@[simp] theorem symm_symm (h : α ≃ₜ β) : h.symm.symm = h := rfl
```

Comment:
This will need an `#align` once the backport goes through. It might be sensible to add it now so we don't forget.